Bio/SeqIO/SwissIO.py

```diff
@@ -73,77 +74,76 @@ def SwissIterator(handle):
-            if swiss_record.host_organism:
-                annotations["organism_host"] = swiss_record.host_organism
-            if swiss_record.host_taxonomy_id:
-                annotations["host_ncbi_taxid"] = swiss_record.host_taxonomy_id
-            if swiss_record.comments:
-                annotations["comment"] = "\n".join(swiss_record.comments)
-            if swiss_record.references:
-                annotations["references"] = []
-                for reference in swiss_record.references:
-                    feature = SeqFeature.Reference()
-                    feature.comment = " ".join(
-                        "%s=%s;" % k_v for k_v in reference.comments
-                    )
-                    for key, value in reference.references:
-                        if key == "PubMed":
-                            feature.pubmed_id = value
-                        elif key == "MEDLINE":
-                            feature.medline_id = value
-                        elif key == "DOI":
-                            pass
-                        elif key == "AGRICOLA":
-                            pass
-                        else:
-                            raise ValueError("Unknown key %s found in references" % key)
-                    feature.authors = reference.authors
-                    feature.title = reference.title
-                    feature.journal = reference.location
-                    annotations["references"].append(feature)
-            if swiss_record.keywords:
-                record.annotations["keywords"] = swiss_record.keywords
-            yield record
+    for swiss_record in swiss_records:
+        # Convert the SwissProt record to a SeqRecord
+        seq = Seq.Seq(swiss_record.sequence, Alphabet.generic_protein)
+        record = SeqRecord.SeqRecord(
+            seq,
+            id=swiss_record.accessions[0],
+            name=swiss_record.entry_name,
+            description=swiss_record.description,
+            features=swiss_record.features,
+        )
+        record.description = swiss_record.description
+        for cross_reference in swiss_record.cross_references:
+            if len(cross_reference) < 2:
+                continue
+            database, accession = cross_reference[:2]
+            dbxref = "%s:%s" % (database, accession)
+            if dbxref not in record.dbxrefs:
+                record.dbxrefs.append(dbxref)
+        annotations = record.annotations
+        annotations["accessions"] = swiss_record.accessions
+        if swiss_record.protein_existence:
+            annotations["protein_existence"] = swiss_record.protein_existence
+        if swiss_record.created:
+            annotations["date"] = swiss_record.created[0]
+            annotations["sequence_version"] = swiss_record.created[1]
+        if swiss_record.sequence_update:
+            annotations["date_last_sequence_update"] = swiss_record.sequence_update[
+                0
+            ]
```

Comment:
Your friend Travis says:

```
$ flake8
./Bio/SeqIO/SwissIO.py:105:85: BLK100 Black would make changes.
```

I think black would put this all on one line now (which is going to look nicer).

There are a couple of places further down which might also get reformatted to use the extra four characters from the indentation change.